pkg/scanner/vulnerability/aqua/converter.go

```diff
@@ -33,7 +33,7 @@ func (c *converter) Convert(in io.Reader) (starboard.VulnerabilityReport, error)
 }
 
 func (c *converter) convert(aquaReport ScanReport) (starboardReport starboard.VulnerabilityReport) {
-	var items []starboard.VulnerabilityItem
+	items := make([]starboard.VulnerabilityItem, 0)
```

Comment:
Nit (which you are welcome to ignore!) - personally I marginally prefer simple declaration rather than zero-length make for slices, because it's one fewer allocation. That is absolutely negligible here so really not important!

Reply:
Yes. I agree. I did that only for the side effect of JSON encoder, i.e. to serialize `vulnerabilities` property to `[]` instead of `null`. Otherwise OpenAPI schema validation that we have for `vulnerabilities.aquasecurity.github.io` would fail.